examples/server/server.cpp

```diff
@@ -3210,7 +3210,8 @@ int main(int argc, char **argv)
                 res.set_content(models.dump(), "application/json; charset=utf-8");
             });
 
-    const auto chat_completions = [&llama, &validate_api_key, &sparams](const httplib::Request &req, httplib::Response &res)
+    const std::string completion_id = gen_chatcmplid();
```

Comment:
This seems to make all responses have the same id.

Don't we want to have a separate id for separate streams?

Reply:
Yes, I messed up 😞 Can confirm that [aa95dc5](https://github.com/ggerganov/llama.cpp/pull/5880/commits/aa95dc5568f9a27f17816426c3a8c118fc7789e3) fixes this up